### DISCUSSION TITLE
Latest Bromite/Chromium for Android KitKat

### DISCUSSION THREAD

**Discussion by sam-gq** [bromite/bromite#1467] [General]
Can you *please* tell me the latest version of Bromite or Chromium for Android KitKat? I couldn't find where the recent version is not compatible🥺🙏🙏. And not sure where the last version from 80-88.

**Comment by Sunspark-007** [comment]
Version 81 is the last officially supported version.

**Reply by sam-gq** [reply]
Ok, I got the most stable v81 already for so long. Thanks!
